data/mods/Xedra_Evolved/spells/XAEA.json

```
@@ -349,14 +349,14 @@
     "id": "XAEA_mod_moves",
     "type": "SPELL",
     "name": { "str": "Artifact Add Moves", "//~": "NO_I18N" },
-    "description": { "str": "Gives 15 - 45 seconds of free actions.", "//~": "NO_I18N" },
+    "description": { "str": "Gives 5 - 15 seconds of free actions.", "//~": "NO_I18N" },
     "valid_targets": [ "self" ],
     "flags": [ "SILENT", "RANDOM_DAMAGE", "IGNORE_WALLS", "RANDOM_DURATION", "NO_EXPLOSION_SFX" ],
     "effect": "mod_moves",
     "shape": "blast",
     "damage_type": "bash",
-    "min_damage": 1500,
-    "max_damage": 4500,
+    "min_damage": 500,
+    "max_damage": 1500,
```

i am not sure i like this numbers, i feel 5 seconds won't do you enough to save you; my quick investigation (i dropped character from a first floor a few times) shows that part of a penalty happens not from decreased speed, but from an increased move cost also.
let's make it this way: 15-25 seconds, but increase the bonus the less speed you have, ie twice as fast for 50 speed
```suggestion
    "min_damage": { "math": [ "1500 / u_speed() * 100" ] },
    "max_damage": { "math": [ "2500 / u_speed() * 100" ] },
```